src/styles/catalog.scss

```
@@ -0,0 +1,102 @@
+.catalog {
+  padding-top: 50px;
+
+  &__container {
+    max-width: 944px;
+    margin: 0 auto;
+  }
+
+  &__row {
+    display: flex;
+    flex-wrap: wrap;
+  }
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  margin-bottom: 30px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__picture {
+    text-align: center;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 37px;
+  }
+
+  &__title {
+    font-weight: 500;
+    line-height: 18px;
+    color: $font-color-dark;
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $font-color-grey;
+  }
+
+  &__review {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $font-color-dark;
+  }
+
+  &__rating {
+    @include flexbox-between;
+    margin-bottom: 24px;
+  }
+
+  &__row {
+    @include flexbox-between;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $font-color-grey;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $font-color-dark;
+  }
+
+  &__link {
+    display: inline-block;
+    margin-bottom: 16px;
+    padding: 11px 54px;
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+    text-transform: uppercase;
+    line-height: 16px;
+    border-radius: 5px;
+    background-color: #00acdc;
+    border: 1px solid transparent ;
+
+    &:hover {
+      border: 1px solid #00acdc;
```

Don't add new border to the element on hover. Add default transparent border of the same width(you already had it in line 94), and change its color on :hover
```suggestion
      border-color: #00acdc;
```

---

+